openupgrade_scripts/scripts/base/14.0.1.3/post-migrate.py

```diff
@@ -143,3 +143,12 @@ def migrate(env, version):
     binary_conversion_for_db_attachments(env)
     # Load noupdate changes
     openupgrade.load_data(env.cr, "base", "14.0.1.3/noupdate_changes.xml")
+    # By default Odoo grants new users export access. If we had web_disable_export_group
+    # installed is very likely that this isn't the desired behavior and that we want
+    # to keep the access to that feature restricted by default.
+    if not env.ref(
+        "web_disable_export_group.group_export_data", raise_if_not_found=False
```

Comment:
AFAIK, this is always going to be False, as you rename it in pre-migration.

Reply:
Ah, better use https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L2542-L2552 then.